Formula/liblxi.rb

```diff
@@ -0,0 +1,43 @@
+class Liblxi < Formula
+  desc "Simple C API for communicating with LXI compatible instruments"
+  homepage "https://github.com/lxi-tools/liblxi"
+  license "BSD-3-Clause"
+  head "https://github.com/lxi-tools/liblxi.git", branch: "master"
+
+  stable do
+    url "https://github.com/lxi-tools/liblxi/archive/refs/tags/v1.18.tar.gz"
+    sha256 "ebeb8cb51f1024e198b3672aadaba113849d20b26199ae6e4ee679e555e9f274"
+    depends_on :linux
+  end
+
+  depends_on "meson" => :build
+  depends_on "ninja" => :build
+  depends_on "pkg-config" => :build
+
+  on_linux do
+    depends_on "libpthread-stubs"
+    depends_on "libtirpc"
+    depends_on "libxml2"
+  end
+
+  def install
+
+    system "meson", "setup", "build", *std_meson_args
+    system "meson", "compile", "-C", "build", "--verbose"
+    system "meson", "install", "-C", "build"
+  end
+
+  test do
+    (testpath/"test.c").write <<~EOS
+      #include <lxi.h>
+      #include <stdio.h>
+
+      int main() {
+        return lxi_init();
+      }
+    EOS
+
+    system ENV.cc, "test.c", "-I#{include}", "-L#{lib}", "-llxi", "-o", "test"
```

Comment:
This will fix your `brew test` failure.
```suggestion
    system ENV.cc, "test.c", "-I#{include}", "-L#{lib}", "-llxi",
                   "-L#{Formula["libtirpc"].opt_include}", "-ltirpc", "-o", "test"
```